### PR TITLE
Minor improvements in peer-to-peer access test

### PR DESCRIPTION
**Pull request by e-kayrakli**
Does the following changes to `nvlink` test:

- Changes the output to say "Bandwidth" instead of "Times"
- Drops the use of a deprecated IO feature
- Moves it to `studies/bandwidth/p2p`

Test:
- [x] amd (before moving the test)
- [x] nvidia

### CODE REVIEW

**Comment by e-kayrakli** [comment]
Marked as a draft to avoid clicking "Merge" during the freeze.

**Review by stonea — APPROVED** [review]

File: test/gpu/native/nvlink.chpl [hunk 1]
```diff
@@ -39,10 +39,10 @@ proc medianTime(times, i, j) {
 }
 
 proc doPrintTimes(times) {
-  writef("Times (GiB/s):\n");
+  writef("Bandwidth (GiB/s):\n");
```
[the remark this answers is not in the document]
Comment (e-kayrakli) [reply test/gpu/native/nvlink.chpl]: https://github.com/chapel-lang/chapel/pull/23538/commits/1e00d9f736aae209ef7904dd050fba57b345a038

**Comment by e-kayrakli** [comment]
@stonea -- I re-asked for review because I move  `nvlink.*` into `studies/bandwidth/p2p.*` as this is not really just about nvlink anymore and that I want to reduce the clutter in `gpu/native`. Let me know if you have any problem with that.

**Review by stonea — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**